src/panels/config/entity_registry/ha-config-entity-registry.ts

```diff
@@ -80,7 +85,19 @@ class HaConfigEntityRegistry extends LitElement {
             </a>
           </span>
           <ha-card>
+            <paper-item>
+              <ha-switch
+                ?checked=${this._showDisabled}
+                @change=${this._showDisabledChanged}
+                >${this.hass.localize(
+                  "ui.panel.config.entity_registry.picker.show_disabled"
+                )}</ha-switch
+              ></paper-item
+            >
             ${this._entities.map((entry) => {
+              if (!this._showDisabled && entry.disabled_by) {
```

Comment:
Instead of doing this inside the map, should we use `.filter` and potentially store that inside a memoize?